src/r_repl/repl.rs

```diff
@@ -54,7 +54,10 @@ where
                         let res = global_env.eval(expr);
                         match res {
                             Ok(val) => println!("{}", val),
-                            Err(e) => println!("{}", e),
+                            Err(e) => match e {
+                                RSignal::Condition(Cond::Terminate) => break,
+                                ref other_error => println!("{}", other_error),
+                            },
```

Comment:
For some reason, my comment with suggestions isn't showing up. What I thought I wrote in earlier was:

```suggestion
                            Err(RSignal::Condition(Cond::Terminate)) => break,
                            Err(e) => println!("{}", e),
```

Reply:
Thanks @dgkf, looks much nicer. I did not know that one can have multiple `Err()` arms. Please consider that latest commit. GitHub did not allow me to apply your suggestion directly in UI since currently they can not apply a suggestion on a delete lines.